### PR TITLE
chore(notes): format release notes using uls

### PR DESCRIPTION
**Pull request by redallen**

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,118 +1,157 @@
...
+- **Dropdown:**
+  - Added space-between when width of dropdown grows ([#2050](https://github.com/patternfly/patternfly-next/pull/2050))
+  - Centered plain button content ([#2071](https://github.com/patternfly/patternfly-next/pull/2071))
+- **Icons:** Changed selector that wraps extend for lower specificity ([#2018](https://github.com/patternfly/patternfly-next/pull/2018))
+- **Login:** Unset the link color text so that its white. ([#2032](https://github.com/patternfly/patternfly-next/pull/2032))
+- **Navigation:** Changed max-height to 100% for subnav  ([#2061](https://github.com/patternfly/patternfly-next/pull/2061))
+- **Options menu:** Centered plain button content ([#2071](https://github.com/patternfly/patternfly-next/pull/2071))
+- **Page:** Removed toggle from horizontal nav page demo ([#2004](https://github.com/patternfly/patternfly-next/pull/2004))
+- **Pagination:** Added disabled variation ([#2015](https://github.com/patternfly/patternfly-next/pull/2015))
+- **Select**
```
Comment (mcoker) [inline RELEASE-NOTES.md]:
```suggestion
- **Select:**
```

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,118 +1,157 @@
...
+- **Table:**
+  - Added an extra breakpoint. In the data table sortable demo the table needed a larger breakpoint, so that the rows don't overflow outside of their container ([#1880](https://github.com/patternfly/patternfly-next/pull/1880))
+  - Added word-wrap to td's in table, now wraps when on mobile size. ([#1928](https://github.com/patternfly/patternfly-next/pull/1928))
+  - Made column headers bold ([#1949](https://github.com/patternfly/patternfly-next/pull/1949))
+## Other
+- Added ie11 to build ([#1876](https://github.com/patternfly/patternfly-next/pull/1876))
+- Updated the window size used for the browser that our a11y audit is run against ([#1911](https://github.com/patternfly/patternfly-next/pull/1911))
 
-# PatternFly RC2.1 (2019-06-11)
+## 2019.02 RC2.1 release notes (2019-06-11)
```
Comment (mcoker) [inline RELEASE-NOTES.md]:
```suggestion
# 2019.02 RC2.1 release notes (2019-06-11)
```

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,118 +1,157 @@
...
+- **Inline alert:**
+  - We had previously applied `font-size` to the inline alerts via `.pf-c-alert__icon > i`, which didn’t work in React since the icon is an svg. We applied `font-size` to `.pf-c-alert__icon` instead. Also, the inline warning icon was fixed because it was wider than the others ([#1909](https://github.com/patternfly/patternfly-next/pull/1909))
+- **Notification Badge:**
+  - Added notification badge ([#1862](https://github.com/patternfly/patternfly-next/pull/1862))
+- **Switch:**
+  - Added focus indicator [(#1874](https://github.com/patternfly/patternfly-next/pull/1874))
+- **Table:**
+  - Media query now matches border. Fixed the table so that when the table breaks to grid form, the border width changes to 8px, and the border color matches the background color. ([#1Packages released:patternfly-next/pull/1881))
+  - Added an Packages released:ithub.com/patternfly/patternfly-next/pull/1880))
+## Docs
```
Comment (mcoker) [inline RELEASE-NOTES.md]: Looks like something went awry in this section.

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,118 +1,157 @@
...
+  - Added breakpoints for actions ([#1886](https://github.com/patternfly/patternfly-next/pull/1886))
+- **Inline alert:**
+  - We had previously applied `font-size` to the inline alerts via `.pf-c-alert__icon > i`, which didn’t work in React since the icon is an svg. We applied `font-size` to `.pf-c-alert__icon` instead. Also, the inline warning icon was fixed because it was wider than the others ([#1909](https://github.com/patternfly/patternfly-next/pull/1909))
+- **Notification Badge:**
+  - Added notification badge ([#1862](https://github.com/patternfly/patternfly-next/pull/1862))
+- **Switch:**
+  - Added focus indicator [(#1874](https://github.com/patternfly/patternfly-next/pull/1874))
+- **Table:**
+  - Media query now matches border. Fixed the table so that when the table breaks to grid form, the border width changes to 8px, and the border color matches the background color. ([#1Packages released:patternfly-next/pull/1881))
+  - Added an Packages released:ithub.com/patternfly/patternfly-next/pull/1880))
```
Comment (mcoker) [inline RELEASE-NOTES.md]: error here

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -1,118 +1,157 @@
...
+- **Datalist:**
+  - Added breakpoints for actions ([#1886](https://github.com/patternfly/patternfly-next/pull/1886))
+- **Inline alert:**
+  - We had previously applied `font-size` to the inline alerts via `.pf-c-alert__icon > i`, which didn’t work in React since the icon is an svg. We applied `font-size` to `.pf-c-alert__icon` instead. Also, the inline warning icon was fixed because it was wider than the others ([#1909](https://github.com/patternfly/patternfly-next/pull/1909))
+- **Notification Badge:**
+  - Added notification badge ([#1862](https://github.com/patternfly/patternfly-next/pull/1862))
+- **Switch:**
+  - Added focus indicator [(#1874](https://github.com/patternfly/patternfly-next/pull/1874))
+- **Table:**
+  - Media query now matches border. Fixed the table so that when the table breaks to grid form, the border width changes to 8px, and the border color matches the background color. ([#1Packages released:patternfly-next/pull/1881))
```
Comment (mcoker) [inline RELEASE-NOTES.md]: error in the link

File: RELEASE-NOTES.md [hunk 6]
```diff
@@ -1,118 +1,157 @@
...
-### Wizard
-*  Fixed wizard closing tag ([#1803](https://github.com/patternfly/patternfly-next/pull/1803))
-*  Set pf-c-wizard__main to grow and fill available height ([#1781](https://github.com/patternfly/patternfly-next/pull/1781))
+- **Alert:**
+  - Adjusted warning alert webfont icon font-size ([#1805](https://github.com/patternfly/patternfly-next/pull/1805))
+  - Added in-line modifier ([#1775](https://github.com/patternfly/patternfly-next/pull/1775))
+- **Card:**
+  - Added a wrapper for the Actions in the top right, so that content wraps around it - similar to the Popover component. Added a logo/img wrapper that is always left aligned. ([#1745](https://github.com/patternfly/patternfly-next/pull/1745))
+  - Removed flex from header ([#1817](https://github.com/patternfly/patternfly-next/pull/1817))
+  - Updated font sizes. Card body and footer text size were changed to be 14px by default and header was changed to16px. ([#2103](https://github.com/patternfly/patternfly-next/pull/2103))
```
Comment (mcoker) [inline RELEASE-NOTES.md]:
```suggestion
  - Updated font sizes. Card body and footer text size were changed to be 14px by default and header was changed to 16px. ([#2103](https://github.com/patternfly/patternfly-next/pull/2103))
```

**Comment by redallen** [comment]
@mcoker Good catches, updated based on feedback.

**Pull request merged**